ast/policy.go

```diff
@@ -154,10 +155,16 @@ type (
 
 	// Annotations represents metadata attached to other AST nodes such as rules.
 	Annotations struct {
-		Location *Location           `json:"-"`
-		Scope    string              `json:"scope"`
-		Schemas  []*SchemaAnnotation `json:"schemas,omitempty"`
-		node     Node
+		Location         *Location                    `json:"-"`
+		Scope            string                       `json:"scope"`
+		Title            string                       `json:"title,omitempty"`
+		Description      string                       `json:"description,omitempty"`
+		Organizations    []string                     `json:"organizations,omitempty"`
+		RelatedResources []*RelatedResourceAnnotation `json:"related_resources,omitempty"`
+		Authors          []*AuthorAnnotation          `json:"authors,omitempty"`
+		Schemas          []*SchemaAnnotation          `json:"schemas,omitempty"`
+		Custom           []*CustomAnnotation          `json:"custom,omitempty"`
```

Comment:
Should this perhaps not be an array, and just be a `map[string]interface{}`; so that the same key cannot be provided multiple times?

Reply:
On further contemplation, I think I should make that change. When we later introduce the `self` keyword, we don't want you need to do
```rego
x := self.metadata.custom[_].foo
```
Also, by making this a YAML map, there is no doubt that multiple values for a single key is somehow supported.